styles/welcome.css

```diff
@@ -0,0 +1,92 @@
+body {
+  font-family: Arial, sans-serif;
```

Comment:
Why Arial? I don't think we should specify font. Have we modeled this after some other extension HTML?